pallets/runtime/tests/src/asset_test.rs

```diff
@@ -1520,27 +1589,35 @@ fn classic_ticker_register_works() {
 
 #[test]
 fn classic_ticker_no_such_classic_ticker() {
+    let user = AccountKeyring::Alice.public();
+    let ticker_acme = ticker("ACME");
+    let ticker_emca = ticker("EMCA");
```

Comment:
Yep, you need to take more time to see the diff.

Reply:
Not sure what you mean by that

Reply:
This improves readability (at least for me) because the "typo" is explicitly expressed in separate objects instead of injected directly inside the function parameters.

Reply:
I'm not sure what you mean by "typo". For me this harms readability. Now instead, I have to track uses of bindings and state rather than immediately concluding "oh, here's some random ticker, name is not important". Temporary bindings in general are not helpful in my view. But mostly, I don't think making these small kinds of changes which aren't obviously positive is a good use of review time.

Reply:
IMO It easier and more readable if you have both ticker in a consecutive lines, instead of having them separated by 20 lines.